std_msgs/Header header

float64 latitude

float64 longitude

float64 altitude

int64 utm_easting

int64 utm_northing

int64 zone

string letter
